package.json

```diff
@@ -35,14 +35,14 @@
   },
   "dependencies": {
     "@angular-redux/store": "^6.4.1",
-    "@angular/common": "2.4.5",
-    "@angular/compiler": "2.4.5",
-    "@angular/core": "2.4.5",
-    "@angular/forms": "2.4.5",
-    "@angular/http": "2.4.5",
-    "@angular/platform-browser": "2.4.5",
-    "@angular/platform-browser-dynamic": "2.4.5",
-    "@angular/router": "3.0.0",
+    "@angular/common": "4.1.3",
```

Comment:
@igor-ka I would rather lock everything down, just because I don't see any benefit to the opposite. Any opinion?